core/math/src/main/java/org/arakhne/afc/math/geometry/d2/afp/Ellipse2afp.java

```diff
@@ -56,6 +56,11 @@
 		B extends Rectangle2afp<?, ?, IE, P, V, B>>
 		extends RectangularShape2afp<ST, IT, IE, P, V, B> {
 
+    final String ELLIPSE_WIDTH_POSITIVE_OR_ZERO = "Ellipse width must be positive or zero";
```

Comment:
In order to share these messages in all the classes for subclasses of Shape2afp, Shape2ai, what do you think to create the `org.arakhne.afc.math.geometry.AssertionMessages` that contains all the message constants for all the shape types?